src/Sylius/Component/Core/Model/CustomerInterface.php

```diff
@@ -61,16 +61,6 @@ public function hasAddress(AddressInterface $address);
     public function getAddresses();
 
     /**
-     * @return BaseUserInterface
-     */
-    public function getUser();
-
-    /**
-     * @param BaseUserInterface|null $user
-     */
-    public function setUser(BaseUserInterface $user = null);
```

Comment:
We could mention deleting these methods in `UPGRADE` file.

Reply:
It's the part of `UserAwareInterface`.